src/main/java/pl/edu/pw/elka/paprykaisalami/geeruh/issues/domain/models/Issue.java

```diff
@@ -18,6 +18,6 @@ public class Issue {
     @NonNull
     Summary summary;
 
-    @Nullable
+    @NonNull
```

Comment:
If you changed this to `NonNull`, you have to also change it in facade, because now providing a null description should raise an exception here. It doesn't because Hibernate implicitly converts null into an empty string when saving to db.

Just pass empty string in the facade if the description in request is null, instead of passing null.

Reply:
Is this what you meant? https://github.com/papryka-i-salami/geeruh/pull/9/files#diff-aef8b7577671590bf597ea85481db33ef06ee252a27bcc0f8fd90c20a3cad9a0R55